### DISCUSSION TITLE
what does 'sip' as in 'isSipScala' or 'inSipScala' stand for?

### DISCUSSION THREAD

**Discussion by mtk** [VirtusLab/scala-cli#762] [General]
subject says it all.

**Comment by SethTisue** [comment]
Scala Improvement Process

see https://contributors.scala-lang.org/t/pre-sip-scala-cli-as-new-scala-command/5628
and https://github.com/VirtusLab/scala-cli/issues/559 for context

basically scala-cli includes core commands and more experimental additional commands, and it's an open question if they all remain just thrown together in one bucket, or what
